### ISSUE TITLE
Modal/dialog should be closed with Esc key

### ISSUE THREAD

**Issue by jrobind** [jrobind/dev-jot#7]
Currently when you view a lesson

![Screen Shot 2020-12-03 at 23 09 18](https://user-images.githubusercontent.com/30320536/101099471-98e1f100-35bc-11eb-98b4-274de034152f.png)

The only way to exit/close the modal is to click the close button. The modal should be closable with the Esc key which will improve modal accessibility for keyboard users.

**Comment by sonechca** [comment]
Hi, 

I would like to work on this, Can I join in this issue?

**Comment by jrobind** [comment]
Go for it @sonechca 😄

**Comment by buigabor** [comment]
I believe it would be also more convenient, if we could close the modal by clicking on the black background as well.

**Comment by jrobind** [comment]
Agreed! Feel free to raise an issue for this @buigabor 😄

**Comment by jrobind** [comment]
I'm going to open this one back up. Let me know if you're still working on this @sonechca!

**Comment by gaddscientist** [comment]
Hi, I'd like to work on this issue. Is it alright to take this one if its back open?

**Comment by jrobind** [comment]
All yours Bryan 👍

**Comment by jrobind** [comment]
Closed by #108
